CE09OSSM/CE09OSSM_D00001_ingest.csv

```diff
@@ -1,19 +1,21 @@
 uframe_route,filename_mask,reference_designator,data_source,
 Ingest.mopak-o-dcl_telemetered,/omc_data/whoi/OMC/CE09OSSM/D00001/cg_data/dcl11/mopak/*.mopak.log,CE09OSSM-SBD11-01-MOPAK0000,telemetered,
+#Ingest.hyd-o-dcl_telemetered,/omc_data/whoi/OMC/CE09OSSM/D00001/cg_data/dcl11/hyd1/*.hyd*.log,CE09OSSM-SBD11-02-?HYD?0000,telemetered,"Hydrogen gas monitoring, uncertain as to Reference Designator"
 Ingest.velpt-ab-dcl_telemetered,/omc_data/whoi/OMC/CE09OSSM/D00001/cg_data/dcl11/velpt1/*.velpt*.log,CE09OSSM-SBD11-04-VELPTA000,telemetered,
 Ingest.metbk-a-dcl_telemetered,/omc_data/whoi/OMC/CE09OSSM/D00001/cg_data/dcl11/metbk/*.metbk.log,CE09OSSM-SBD11-06-METBKA000,telemetered,
+#Ingest.hyd-o-dcl_telemetered,/omc_data/whoi/OMC/CE09OSSM/D00002/cg_data/dcl12/hyd2/*.hyd*.log,CE09OSSM-SBD12-03-?HYD?0000,telemetered,"Hydrogen gas monitoring, uncertain as to Reference Designator"

```

Comment:
Reference designator should be CE09OSSM-SBD12-03-HYDGN0000 and and file path should include D00001 rather than D00002. With these changes, we can remove the # at the start of the line.